diktat-maven-plugin/src/test/kotlin/org/cqfn/diktat/plugin/maven/DiktatMavenPluginIntegrationTest.kt

```diff
@@ -5,60 +5,105 @@ import com.soebes.itf.jupiter.extension.MavenJupiterExtension
 import com.soebes.itf.jupiter.extension.MavenTest
 import com.soebes.itf.jupiter.maven.MavenExecutionResult
 import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.TestInfo
 import java.io.File
 import kotlin.io.path.ExperimentalPathApi
+import kotlin.io.path.Path
+import kotlin.io.path.deleteIfExists
+import kotlin.io.path.div
 import kotlin.io.path.readText
 
 /**
  * Integration tests for diktat-maven-plugin. Run against the project from diktat-examples.
  * The whole pipeline is as follows:
- * * For each test case, test data is copied from examples with respect to maven-itf requirements, .mvn/jvm.config and .mvn/maven.config are copied too
- *   Note: for maven itf test name should equal example project's directory name, which we have in pom.xml.
- * * maven-failsafe-plugin launches tests; for each test case a separate maven process is spawned. diktat.version is taken from .mvn/maven.config
+ * * For each test case, test data is copied from examples with respect to maven-itf requirements, .mvn/jvm.config and .mvn/maven.config are copied, too.
+ *   **Note**: for maven its test name should equal example project's directory name, which we have in `pom.xml`.
+ * * maven-failsafe-plugin launches tests; for each test case a separate maven process is spawned. `diktat.version` is taken from `.mvn/maven.config`
  *   and the exact value is written when maven copies resources.
- * * maven execution results are analyzed here; .mvn/jvm.config is used to attach jacoco java agent to every maven process and generate individual execution reports
+ * * maven execution results are analyzed here; `.mvn/jvm.config` is used to attach jacoco java agent to every maven process and generate individual execution reports.
+ *
+ * When within an IDE (e.g.: _IDEA_), don't run this test directly: it's
+ * expected to be executed from a forked JVM. Instead, run a `verify` _lifecycle
```

Comment:
Did you have a chance to try to run this forked JVM with remote debugging support? Such instructions would be useful here too

Reply:
Yeah, some notes about the [remote debugger](https://www.jetbrains.com/help/idea/tutorial-remote-debug.html#990f7e4d) in the case of diktat will be great here.

But actually I love ultimate debugging tool called `print`

Reply:
It's on my TODO list, will update the KDoc.